### ISSUE TITLE
Panics on gstreamer-rs workspace "BUG: update() must be called before query()"

### ISSUE THREAD

**Issue by sdroege** [kbknapp/cargo-outdated#91]
Clone the repository at https://github.com/sdroege/gstreamer-rs, run cargo update in the root and then run cargo outdated. It fails like below.

This is with cargo 0.22 and cargo outdated 0.6.0.

```
$ env RUST_BACKTRACE=1 cargo outdated -w
thread 'main' panicked at 'BUG: update() must be called before query()', /checkout/src/libcore/option.rs:819:4
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:71
             at /checkout/src/libstd/sys_common/backtrace.rs:60
             at /checkout/src/libstd/panicking.rs:381
   2: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:397
             at /checkout/src/libstd/panicking.rs:611
   3: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:572
   4: std::panicking::begin_panic_fmt
             at /checkout/src/libstd/panicking.rs:522
   5: core::panicking::panic_fmt
             at /checkout/src/libstd/panicking.rs:498
   6: core::option::expect_failed
             at /checkout/src/libcore/option.rs:819
   7: <cargo::sources::git::source::GitSource<'cfg> as cargo::core::registry::Registry>::query
   8: cargo::core::registry::Registry::query_vec
   9: cargo_outdated::cargo_ops::temp_project::TempProject::find_update
  10: cargo_outdated::cargo_ops::temp_project::TempProject::update_version_and_feature
  11: cargo_outdated::execute
  12: cargo_outdated::main
  13: main
  14: __libc_start_main
  15: _start
```

**Comment by Frederick888** [comment]
Should be fixed by #93

**Comment by sdroege** [comment]
Thanks, works! 🎆
